rest-api/alembic/versions/671f14f9e358_add_ehr_metric_fields.py

```diff
@@ -0,0 +1,57 @@
+"""add ehr metric fields
+
+Revision ID: 671f14f9e358
+Revises: fac398dd2426
+Create Date: 2019-02-20 09:38:20.642286
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import model.utils
+
+
+from participant_enums import EhrStatus
+
+# revision identifiers, used by Alembic.
+revision = '671f14f9e358'
+down_revision = 'fac398dd2426'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('participant_summary', sa.Column('ehr_receipt_time', model.utils.UTCDateTime(), nullable=True))
+    op.add_column('participant_summary', sa.Column('ehr_status', model.utils.Enum(EhrStatus), nullable=True))
+    op.add_column('participant_summary', sa.Column('ehr_update_time', model.utils.UTCDateTime(), nullable=True))
+    # ### end Alembic commands ###
```

Comment:
Should this migration also replace the existing `participant_view` to expose these new fields?